go.mod

```diff
@@ -6,6 +6,23 @@ replace github.com/hashicorp/boundary/api => ./api
 
 replace github.com/hashicorp/boundary/sdk => ./sdk
 
+replace (
```

Comment:
This is needed due to the warning shown here: https://golang.testcontainers.org/quickstart/gotest/

This should be able to be removed in the next version of testcontainers.